extensions/markdown-language-features/src/util/topmostLineMonitor.ts

```diff
@@ -7,13 +7,21 @@ import * as vscode from 'vscode';
 import { Disposable } from '../util/dispose';
 import { isMarkdownFile } from './file';
 
+export interface LastScrollLocation {
+	readonly line: number;
+	readonly uri: vscode.Uri | undefined;
```

Comment:
What are the cases where uri can be undefined? Is this something we'd expect to happen?

Reply:
Originally allowed for undefined for initialization; removed undefined.